src/Tests/RepoSyncTests.cs

```diff
@@ -13,7 +13,10 @@ public async Task SyncPrIncludeAllByDefault()
         repoSync.RemoveBlob("README.md");
         repoSync.AddTargetRepository(new(credentials, Client.RepositoryOwner, "GitHubSync.TestRepository", repoContext.TempBranchName));
 
-        var sync = await repoSync.Sync();
+        var sync = await repoSync.Sync(
```

Comment:
shouldnt these be in new tests?

Reply:
do you mean to make the `GitHubSync*` messages the defaults for prTitle, branchName, & commitMessage? or create new tests which don't use the previous defaults?